Automattic/jetpack
# Enable multiple sharing buttons

Fixes https://github.com/Automattic/jetpack/issues/35578

## Proposed changes:

After this fix: https://github.com/Automattic/jetpack/pull/35382 we are able to enable multiple sharing button blocks. There's no need for this limitation.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Spin up the branch locally (or JN)
- On your site create a new post. Add multiple instances of Sharing Buttons block and fill with services. On the post, verify that all buttons works correctly.
